server/pgServer.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+const cors = require('cors');
+const expressStaticGzip = require('express-static-gzip');
+const knex = require('../knexConnection.js');
+const bodyParser = require('body-parser');
+const port = 3005;
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+
+// app.use(express.static('client'));
+app.use('/', expressStaticGzip(path.join(__dirname, '../client'), {
+  enableBrotli: true,
+  orderPreference: ['br', 'gz'],
+  setHeaders: function (res, path) {
+    res.setHeader("Cache-Control", "client, max-age=31536000");
+  }
+}));
+
+app.use(cors());
+
+// ---------------- CRUD FOR POSTGRESQL --------------- //
+
+app.get('/songdata/:id', async (req, res) => {
+  try {
+    var id = req.params.id;
+    if (id > 10000001 || id < 0) {
+      throw new Error(`song_id: ${id} does not exist`);
```

Comment:
Instead of throwing an error, have you considered returning 404 Not Found. https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

Reply:
Done!